app/src/components/market/market_home_container.tsx

```diff
@@ -38,7 +40,7 @@ const MarketHomeContainer: React.FC = () => {
   }, [provider])
 
   useEffect(() => {
-    if (loading) {
+    if (loading && moreMarkets) {
```

Comment:
I this necessary? The way I see this, when the last fetch is done:

- fetchMore is called, loading is set to true
- data is returned, loading is set to false (and so this condition is false) and fetched data is []
- since data is [], `moreMarkets` is set to false, but loading is still false so this condition continues to be false

Am I missing something?

Reply:
No, you're right. It's removed now.